sidechain/sidechain-storage/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "its-sidechain-storage"
```

Comment:
As @clangenb has pointed out, I think `its-sidechain-storage` is redundant. `its` stands for `integritee-sidechain`. So we should probably name it `its-storage`. And the corresponding folder should be just `storage`, not `sidechain-storage`.

Reply:
https://github.com/integritee-network/worker/pull/529#discussion_r755894843

> its-sidechain-storage has redundant naming due to its standing for integritee-sidechain already.
> 
> I think we can simply call it its-db

its-db or its-storage?

Reply:
I discussed a bit with Chris - we go with `its-storage` for now.